.github/workflows/docs.yml

```diff
@@ -0,0 +1,51 @@
+name: AMR-Wind Docs
+
+ on:
+   push:
+     branches:
+       - development
+     paths:
+       - 'amr-wind/**'
+       - 'docs/**'
+       - '.github/workflows/docs.yml'
+   pull_request:
+     branches:
+       - development
+     paths:
+       - 'amr-wind/**'
+       - 'docs/**'
+       - '.github/workflows/docs.yml'
+
+ jobs:
+   Docs:
+     runs-on: ubuntu-latest
+     steps:
+       - uses: actions/checkout@v2
+       - name: Python
+         uses: actions/setup-python@v2
+         with:
+           python-version: '3.x'
+       - name: Dependencies
+         run: |
+           export DEBIAN_FRONTEND=noninteractive
+           sudo apt-get update
+           sudo apt-get install -y --no-install-recommends \
+             doxygen graphviz
+           python3 -m pip install --upgrade pip
+           pip3 install sphinx sphinx_rtd_theme
+       - name: Build
+         working-directory: ${{github.workspace}}
```

Comment:
Not necessary

Reply:
Yes not necessary, but as someone who would reference this to build the docs myself, I think it's helpful to see the working directory explicitly.

Reply:
Sure... just requires that person to know whether `github.workspace` refers to the parent directory into which the project is cloned by `actions/checkout@v2` or the root directory of the git repo. 

May be a comment to this effect?

```
# execute from top amr-wind directory
```

Reply:
Heh yeah...